conans/test/integration/command_v2/test_version.py

```diff
@@ -0,0 +1,19 @@
+from conans.test.utils.tools import TestClient
+
+
+def test_version_json():
+    t = TestClient()
+    t.run("version --format=json")
+    assert ['{', '    "version": "2.0.4"', '}'] == t.out.splitlines()
```

Comment:
```suggestion
    t.run("version --format=json")
    js = json.loads(t.stdout)
    assert js["version"] == __version__
```
These tests will fail whenever there is a version bump. They need to be parameterized using the current ``__version__``

Reply:
Fixed. I forgot to replace hardcoded values 🤦